R/load.R

```diff
@@ -64,6 +64,16 @@
 #'    behavior when loading an installed package with [library()], and can
 #'    be useful for checking for missing exports.
 #'
+#' # Controlling the debug compiler flags
+#'
+#' `load_all()` delegates to [pkgbuild::compile_dll()] to perform the actual
+#' compilation, during which by default some debug compiler flags are
+#' appended. If this is undesired (for example, if you would like to produce
+#' an optimized build), you can opt out by setting the `pkg.build_extra_flags`
```

Comment:
```suggestion
#' appended. If you would like to produce an optimized build instead),
#' you can opt out by setting the `pkg.build_extra_flags`
```

Reply:
Thanks for the suggestion! I didn't apply it directly for a mismatched bracket. :)